springboot/src/main/java/ca/jrvs/apps/trading/controller/OrderController.java

```diff
@@ -0,0 +1,43 @@
+package ca.jrvs.apps.trading.controller;
+
+import ca.jrvs.apps.trading.model.domain.MarketDataDto;
+import ca.jrvs.apps.trading.model.domain.SecurityOrder;
+import ca.jrvs.apps.trading.service.OrderService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
```

Comment:
make sure you understand all annotations. I will ask you related questions during the TCR